### ISSUE TITLE
Is it possible to use expo?

### ISSUE THREAD

**Issue by imthedaniels** [thecodingmachine/react-native-boilerplate#437]
### Is your feature request related to a problem?

What would I need to do to use expo in this project?

### Describe the solution you'd like

What would I need to do to use expo in this project?

**Comment by shivkr6** [comment]
YES. It works for me. Just one command away and expo will do everything for you.
`npx install-expo-modules@latest`
https://docs.expo.dev/bare/installing-expo-modules/
